### PR TITLE
[MOD-2182] Update Ti.Admob to work with iOS 8+, update iOS SDK to 7.6.0

### PR DESCRIPTION
**Pull request by hansemannn**
JIRA: https://jira.appcelerator.org/browse/MOD-2182

### CODE REVIEW

**Comment by hansemannn** [comment]
Thanks @stephenfeather! Just corrected the typo, the module should be good to go now.

File: .travis.yml [hunk 1]
```diff
@@ -1,16 +1,19 @@
...
+   - "MODULE_NAME=ti.admob"
 before_install:
     - MODULE_ROOT=$PWD
 install:
     - cd $MODULE_ROOT
-    - curl -o install.sh https://raw.githubusercontent.com/appcelerator-modules/ci/master/travis/install.sh
-    - source install.sh
-script: 
+    - curl -o install.sh https://raw.githubusercontent.com/appcelerator-modules/ci/master/travis/install.sh #change this to appcelerator-modules once PR has been merged
+    - source install.sh -s "--branch 5_1_X"
```
Comment (AngelkPetkov) [inline .travis.yml]: Shouldn't we install the 5_2_X instead ? As ones its merged it will go in to 5_2_X

Comment (hansemannn) [inline .travis.yml]: It's the CI script, which needs to cover the minimum SDK version the module requires (which is 5.1.0)

Comment (AngelkPetkov) [inline .travis.yml]: Ah , okay

File: ios/documentation/view.md [hunk 2]
```diff
@@ -12,6 +12,9 @@ ad will be loaded from the server automatically:
 * 468x60
 * 728x90
 
+Note: Starting in 2.0.0, you just need to specify a height and and apropiate ad is selected by the Admob SDK.
```
Comment (AngelkPetkov) [inline ios/documentation/view.md]: change to appropriate

**Comment by AngelkPetkov** [comment]
FT and CR approved looks good, thanks Hans!

**Pull request merged**